xapian-applications/omega/omindexcheck.cc

```diff
@@ -131,6 +131,44 @@ index_test()
     tests.insert({"sof/libarchive_openoffice_presentation_template.sti",
 		  {"ZSproject", "ZSresearch", "Zhead", "Ztext"}});
 #endif
+#if defined HAVE_LIBARCHIVE
+    // blank file
+    tests.insert({"odf/blank.odt", {}});
+    // corrupted file (ODP)
+    // tests.insert({"corrupt_file.odp", {"ZSnatur"}});
+
+    // ODF
+    tests.insert({"odf/test.odt", {"Zедой", "Z喬伊不分享食物"}});
+    tests.insert({"odf/libarchive_text.odt", {"Stesttitle", "Atestauthor",
+		  "Zsampl", "Zhead", "Ztext", "Zhello",
+		  "Zworld"}});
+    tests.insert({"odf/libarchive_text_template.ott", {"Zjane", "Zdoe",
+		  "Zstructur"}});
+    tests.insert({"odf/libarchive_presentation.odp", {"Zfascin", "Zfact",
+		  "Zpustak", "Zmahal", "Zmillion", "Zpeopl", "Zbirthday",
+		  "501"}});
+    tests.insert({"odf/libarchive_presentation_template.otp", {"ZSalizarin",
+		  "Zhead", "Zworld", "Ztext"}});
+    tests.insert({"odf/libarchive_spreadsheet.ods", {"Zhello", "Zworld",
+		  "Zsampl", "2"}});
+    tests.insert({"odf/libarchive_spreadsheet_template.ots", {"Zfood", "Zpasta",
+		  "Zpercentag", "40"}});
+    tests.insert({"odf/libarchive_draw.odg", {"Zparth", "Zkapadia"}});
+
+    // Apache OpenOffice
+    tests.insert({"sof/libarchive_openoffice_calc.sxc", {"Ztoy", "Zproduct",
+		  "Zcost", "Zquantiti", "Zcardboard"}});
+    tests.insert({"sof/libarchive_openoffice_calc_template.stc", {"ZSpurchas",
+		  "ZStemplat", "Zproduct", "Zquantiti", "Zsampl"}});
+    tests.insert({"sof/libarchive_openoffice_text.sxw", {"Zhello", "Zsampl",
+		  "Zopenoffic", "Zwriter"}});
+    tests.insert({"sof/libarchive_openoffice_text_template.stw", {"Zhello",
+		  "Zworld", "Zsampl", "Zhead", "ZStemplat", "ZStext"}});
+    tests.insert({"sof/libarchive_openoffice_presentation.sxi", {"Zhead",
+		  "Zhello", "Zopenoffic", "Zimpress"}});
+    tests.insert({"sof/libarchive_openoffice_presentation_template.sti",
+		 {"ZSproject", "ZSresearch", "Zhead", "Ztext"}});
+#endif
 }
 
 static bool
```

Comment:
When I did git commit -v, this wasn't there. I even compared before creating a PR. I don't know how this end up here.

What I did was I performed a git reset --mixed on my omindex_mimetype-modifications branch to reset it to a commit that has been merged. And then did git add -p to only add changes which were required.

Reply:
> When I did git commit -v, this wasn't there. I even compared before creating a PR. I don't know how this end up here.
> 
> What I did was I performed a git reset --mixed on my omindex_mimetype-modifications branch to reset it to a commit that has been merged. And then did git add -p to only add changes which were required.

I have fixed this, It was added twice, I will find out how that happened. I have fixed it for this PR.